versioned/tiered/gc/src/test/java/org/projectnessie/versioned/tiered/gc/ITTestIdentifyUnreferencedAssets.java

```diff
@@ -129,9 +122,14 @@ public void run() throws Exception {
         .maxAgeMicros(ONE_DAY_OLD_MICROS)
         .timeSlopMicros(ONE_HOUR_OLD_MICROS)
         .build();
-    IdentifyUnreferencedAssets<DummyValue> app = new IdentifyUnreferencedAssets<DummyValue>(helper, new DynamoSupplier(), spark, options);
-    Dataset<UnreferencedItem> items = app.identify();
-    Set<String> unreferencedItems = items.collectAsList().stream().map(UnreferencedItem::getName).collect(Collectors.toSet());
+    IdentifyUnreferencedValues<DummyValue> app = new IdentifyUnreferencedValues<>(helper, new DynamoSupplier(), spark, options);
+    Dataset<CategorizedValue> values = app.identify();
+
+    IdentifyUnreferencedAssets<DummyValue, DummyAsset> ident = new IdentifyUnreferencedAssets<>(helper.getValueSerializer(),
```

Comment:
weird variable names here (ident) and above (app)

Reply:
fixed